.github/CODEOWNERS

```diff
@@ -4,7 +4,7 @@ po/da.po @mgeisler @voss @thedataking
 po/de.po @ronaldfw @fechu
 po/el.po @Mandragorian
 po/es.po @deavid @JaviSorribes
-po/fa.po @DannyRavi
+po/fa.po @DannyRavi @hamidrezakp @moaminsharifi @mehrdad77
```

Comment:
Can you check the user @mehrdad77? Apparently it's not an existing username.

Reply:
Ops that's my fault, it was `mehrad77`